docs/contributor/releasing.md

```diff
@@ -9,40 +9,15 @@ This release process covers the steps to release new major and minor versions fo
 
 3. Create a new [GitHub milestone](https://github.com/kyma-project/opentelemetry-collector-components/milestones) for the next version.
 
-4. In the `opentelemetry-collector-components` repository, create a release branch.
-   The name of this branch must follow the `release-x.y` pattern, such as `release-1.0`. As opentelemetry-collector follows the release schedule of the `telemetry-manager` component, the release branch should also have the same name as the `telemetry-manager` release branch.
+4. Manually trigger the [Create release](https://github.com/kyma-project/opentelemetry-collector-components/actions/workflows/create-release.yaml) GitHub Action from the target branch and provide an appropriate release tag as an arugment. The target branch must be:
+   - `main` for major and minor releases,
+   - `release-x.y` for patch releases of version x.y.
 
-   ```bash
-   git fetch upstream
-   git checkout --no-track -b {RELEASE_BRANCH} upstream/main
-   git push upstream {RELEASE_BRANCH}
-   ```
+> [!NOTE]
+> - If you check `Do-Not-Publish`, the release will not be published. However, the action still creates an image and a draft release.
+> - If you check `Force`, it overwrites the existing release and any previously built images. It will never overwrite an already published release. For draft releases, the force option: - recreates the Git tag in the repository - recreates the release notes - overwrites the previously built image
```

Comment:
```suggestion
> - If you select `Force`, it overwrites the existing release and any previously built images. It will never overwrite an already published release. For draft releases, the force option recreates the Git tag in the repository, recreates the release notes, and overwrites the previously built image.
```